airbyte-workers/src/main/java/io/airbyte/workers/process/KubeProcessFactory.java

```diff
@@ -70,32 +74,36 @@ public KubeProcessFactory(final String namespace,
    *        itself
    * @param processRunnerHost is the local host or ip of the machine running the process factory.
    *        injectable for testing.
+   * @param isOrchestrator determines if this should run as airbyte-admin
```

Comment:
ah because this needs to create other pods?

Reply:
yep